deny.toml

```diff
@@ -83,7 +83,6 @@ ignore = [
     { id = "RUSTSEC-2021-0153", reason = "`encoding` is used by lindera" },
     { id = "RUSTSEC-2024-0384", reason = "`instant` is used by tantivy" },
     { id = "RUSTSEC-2024-0436", reason = "`paste` is used by datafusion" },
-    { id = "RUSTSEC-2025-0014", reason = "`humantime` is used by object_store" },
```

Comment:
See [RUSTSEC-2025-0014](https://rustsec.org/advisories/RUSTSEC-2025-0014) for reference.